allofplos/transformations.py

```diff
@@ -173,7 +154,7 @@ def doi_to_url(doi):
     :param doi: full unique identifier for a PLOS article
     :return: online location of a PLOS article's XML
     """
-    journal = doi_to_journal(doi)
+    journal = Journal.doi_to_journal(doi)
```

Comment:
If you had the backwards compatibility layer, you wouldn't need to change this API.